tests/restv1/EventUpdateCest.php

```diff
@@ -125,12 +124,14 @@ public function it_should_allow_to_set_the_start_date_using_natural_language( Te
 		$I->generate_nonce_for_role( 'administrator' );
 
 		$start = $data[0];
-		$end = $data[1];
+		$end   = $data[1];
 
-		$start_date = date( 'Y-m-d H:i:s', strtotime( $start ) );
-		$end_date   = date( 'Y-m-d H:i:s', strtotime( $end ) );
+		$timezone   = 'America/New_York';
+		$start_obj  = new DateTime( $start, new DateTimeZone( $timezone ) );
+		$end_obj    = new DateTime( $end, new DateTimeZone( $timezone ) );
```

Comment:
Why not only generate the TZ once:
```suggestion
		$timezone     = 'America/New_York';
		$timezone_obj = new DateTimeZone( $timezone );
		$start_obj  = new DateTime( $start, $timezone_obj );
		$end_obj    = new DateTime( $end, $timezone_obj );
```
Need the `$timezone` string below, otherwise I'd compact that even further 😉

Reply:
Rinse and repeat below as needed

Reply:
This is an optimization of tests, so it's not _needed_ for merging.

Reply:
Fair point. I'll tackle some of these in my next PR so they don't go by the roadside.